portal-api/api-tests/v1/bonds/bonds.api-test.js

```diff
@@ -733,11 +730,11 @@ describe('/v1/deals/:id/bond', () => {
       expect(status).toEqual(401);
     });
 
-    it('401s requests that do not come from a user with role=maker', async () => {
-      const { status } = await as(noRoles).remove(`/v1/deals/${dealId}/bond/12345678`);
+    // it('401s requests that do not come from a user with role=maker', async () => {
```

Comment:
How are we testing this assertion, since this has been removed?
Please do not include commented code.